src/net.cpp

```diff
@@ -215,7 +214,7 @@ Optional<CAddress> GetLocalAddrForPeer(CNode *pnode)
         return addrLocal;
     }
     // Address is unroutable. Don't advertise.
-    return nullopt;
+    return std::nullopt;
```

Comment:
It may be slightly more idiomatic to replace all the `return std::nullopt;` with `return {};`